shell/browser/extensions/electron_component_extension_resource_manager.cc

```diff
@@ -17,7 +17,7 @@
 #include "electron/buildflags/buildflags.h"
 
 #if BUILDFLAG(ENABLE_PDF_VIEWER)
-#include "chrome/browser/pdf/pdf_extension_util.h"
+#include "chrome/browser/pdf/pdf_extension_util.h"  // nogncheck
```

Comment:
Why are we adding these `nogncheck`s?

Reply:
I forgot to add those in the previous PR #25494